roles/core/nfs_client/molecule/default/verify.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: Verify
+  hosts: all
+  tasks:
+
+    - name: Collect package facts
+      package_facts:
+        manager: auto
+
+    - name: Collect services facts
+      service_facts:
+
+    - name: Assert nfs needed packages are installed
+      assert:
+        that: "'nfs-utils' in ansible_facts.packages"
+
+    - name: Retrieve dir system status
+      stat:
+        path: /opt/software
+      register: register_stat
+
+    - name: Assert dir /etc/software exist
+      assert:
+        that: 
+          - "register_stat.stat.exists"
+          - "register_stat.stat.isdir"
+
+    - name: Retrieve /opt/software in /etc/fstab file
+      lineinfile:
```

Comment:
If I remember correctly this module modifies the file.
I would take an approach similar to [the verify of the role time](https://github.com/bluebanquise/bluebanquise/blob/0cfa48ba0e09991753cdbe98f3e1176ac808a295/roles/core/time/molecule/default/verify.yml#L30):
 - read /etc/fstab once
 - assert a line exists with regex_search

Reply:
On second read, I understand the test:

- check for the line with regexp
- remove the line if present
- check mode (let the file untouched)
- test the result

It is somewhat different than what I saw in the past. It works.

Reply:
Yes, I am using the check_mode here.